src/main/java/com/spotinst/sdkjava/example/statefulnode/azure/AzureStatefulNodeExample.java

```diff
@@ -0,0 +1,558 @@
+
+        StatefulNodeDeallocationConfig statefulDeallocationConfig = statefulDeallocationConfigBuilder.setShouldTerminateVms(true)
+                                       .setNetworkDeallocationConfig(networkDeallocationConfig).setDiskDeallocationConfig(diskDeallocationConfig)
+                                       .setSnapshotDeallocationConfig(snapchatDeallocationConfig).setPublicIpDeallocationConfig(publicIpDeallocationConfig).build();
+
+        // Build node deletion request
+        StatefulNodeDeletionRequest.Builder nodeDeletionRequestBuilder = StatefulNodeDeletionRequest.Builder.get();
+        StatefulNodeDeletionRequest         deletionRequest            = nodeDeletionRequestBuilder.setDeallocationConfig(statefulDeallocationConfig).build();
+
+        System.out.println(String.format("Stateful Node deletion Request: " + deletionRequest.toJson()));
+        Boolean deleteNodeResponse = client.deleteNode(deletionRequest,nodeId);
+
+        System.out.println(String.format("Stateful Node %s is successfully Deleted", nodeId));
+        return deleteNodeResponse;
+    }
+
+    private static List<StatefulNode> getAllStatefulNode(SpotinstAzureStatefulNodeClient client) {
+
+        // Get All Stateful Nodes
+        List<StatefulNode> statefulNodesList = client.getAllNodes();
+
+        System.out.println("Stateful Nodes List: ");
+        for (int i=0; i< statefulNodesList.size(); i++){
+            System.out.println(String.format("Stateful Node Id: %s and Name: %s ",statefulNodesList.get(i).getId() , statefulNodesList.get(i).getName()));
+
+        }
+
+        return statefulNodesList;
+
+    }
+
+    private static Boolean recycleStatefulNode(SpotinstAzureStatefulNodeClient client, String nodeId){
+
+        // Recycle Stateful Nodes
+        StatefulNodeStateRequest.Builder recycleStatefulNodeBuilder = StatefulNodeStateRequest.Builder.get();
+        StatefulNodeStateRequest recycleStatefulNode = recycleStatefulNodeBuilder.setState(AzureStatefulNodeStateEnum.RECYCLE).build();
+
+        Boolean recycleStatefulNodeResponse = client.updateStatefulNodeState(recycleStatefulNode,nodeId);
+
+        System.out.println(String.format("Recycle of stateful Node %s is successful ", nodeId));
+        return recycleStatefulNodeResponse;
+
+    }
+
+    private static Boolean pauseStatefulNode(SpotinstAzureStatefulNodeClient client, String nodeId){
+
+        // Recycle Stateful Nodes
+        StatefulNodeStateRequest.Builder pauseStatefulNodeBuilder = StatefulNodeStateRequest.Builder.get();
+        StatefulNodeStateRequest pauseStatefulNode = pauseStatefulNodeBuilder.setState(AzureStatefulNodeStateEnum.PAUSE).build();
+
+        Boolean pauseStatefulNodeResponse = client.updateStatefulNodeState(pauseStatefulNode,nodeId);
+
+        System.out.println(String.format("Pause of stateful Node %s is successful ", nodeId));
```

Comment:
Same comments as previous function. The pause may be unsuccessful.

Reply:
Incorporated. Done